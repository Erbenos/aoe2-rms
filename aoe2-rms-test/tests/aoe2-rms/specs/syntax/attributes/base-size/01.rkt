#lang aoe2-rms
; Create a 13x13 square of ice

<LAND-GENERATION>
(create-land
 (terrain-type 'ICE)
 (base-size 6)
 (number-of-tiles 0))
