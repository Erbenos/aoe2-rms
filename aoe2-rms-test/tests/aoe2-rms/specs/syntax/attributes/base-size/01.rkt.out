<LAND_GENERATION>
create_land {
  terrain_type ICE
  base_size 6
  number_of_tiles 0
}
